docs/canvas/canvas-common-functions.asciidoc

```diff
@@ -6,640 +6,765 @@ The common functions can run anywhere, which means they'll execute wherever
-|string
-|The type to convert the column to; leave blank to not change type (default: `null`)
+|`type`
+|`string`
+|The type to convert the column to; leave blank to not change type
 
-|name
-|string
-|The resultant column name; leave blank to not rename (default: `null`)
+|`name`
+|`string`
+|The resultant column name; leave blank to not rename
 |===
 
-*Returns:* datatable
+*Returns:* `datatable`
 
 
 [float]
-=== any
+[[any_fn]]
+=== `any`
 
-Returns true if any of the conditions are true.
+Returns `true` if any of the conditions are `true`. See also <<all_fn>>.
 
-*Accepts:* Anything or nothing
+*Accepts:* `null`
 
 [cols="3*^<"]
 |===
 |Argument |Type |Description
 
-|_Unnamed_*
-|boolean
+|_Unnamed_ *** *†*
+
+Alias: `condition`
+|`boolean`
 |The conditions to check
 |===
 
-*Returns:* boolean
+*Returns:* `boolean`
 
 
 [float]
-=== as
+[[as_fn]]
+=== `as`
 
-Creates a datatable with a single value.
+Creates a `datatable` with a single value. See also <<getCell_fn>>.
 
-*Accepts:* string, boolean, number, null
+*Accepts:* `string`, `boolean`, `number`, `null`
 
 [cols="3*^<"]
 |===
 |Argument |Type |Description
 
-|_Unnamed_ (Alias: `name`)
-|string
-|A name to give the column (default: `value`)
+|_Unnamed_ 
+
+Alias: `name`
+|`string`
+|A name to give the column 
+
+Default: `"value"`
 |===
 
-*Returns:* datatable
+*Returns:* `datatable`
 
 
 [float]
-=== axisConfig
+[[axisConfig_fn]]
+=== `axisConfig`
 
-Configures axis of a visualization.
+Configures axis of a visualization, only used in <<plot_fn>>
 
-*Accepts:* datatable
+*Accepts:* `null`
 
 [cols="3*^<"]
 |===
 |Argument |Type |Description
 
-|show
-|boolean
-|default: `true`
+|`show`
+|`boolean`
+|Show the axis labels? 
+
+Default: `true`
+
+|`position`
+|`string`
+|The position of the axis labels, for example, top, bottom, left, and right 
+
+Default: `"left"`
+
+|`min`
+|`number`, `string`, `null`
+|Minimum value displayed in the axis. Must be a `number`, a date in milliseconds since epoch, or an ISO8601 `string`
+
+|`max`
+|`number`, `string`, `null`
+|Maximum value displayed in the axis. Must be a `number`, a date in milliseconds since epoch, or an ISO8601 `string`
 
-|position
-|string
-|The position of the axis labels, for example, top, bottom, left, and right
+|`tickSize`
+|`number`, `null`
+|Increment size between each tick. Use for `number` axes only.
 |===
 
-*Returns:* axisConfig
+*Returns:* `axisConfig`
 
 
 [float]
-=== case
+[[case_fn]]
+=== `case`
 
-Builds a case (including a condition/result) to pass to the switch function.
+Builds a `case` (including a condition/result) to pass to the <<switch_fn>> function.
 
-*Accepts:* Anything or nothing
+*Accepts:* `any`
 
 [cols="3*^<"]
 |===
 |Argument |Type |Description
 
-|_Unnamed_
-|any
-|This value is compared to the context to see if the condition 
-s met. It is overridden by the "if" argument if both are provided.
+|_Unnamed_ 
+
+Alias: `when`
+|`any`
+|This value is compared to the _context_ to see if the condition 
```

Comment:
How about:

`To ensure that the condition is met, the `any` value is compared to the `context`. If the `if` argument is provided, the value is overridden.`